doc/rtd/topics/instancedata.rst

```diff
@@ -171,6 +214,14 @@ Examples output:
 
 - ['ssh-rsa AA...', ...]
 
+v1.python
+---------
+The default version of python on the system
```

Comment:
That's not correct; rather this is the version of python _executing_ cloud-init.

Reply:
How about python_version for the key name?

Reply:
Mention this is from util.system_info()

Reply:
+1 to all